### ISSUE TITLE
BUILD FAILED - "option 6 is no longer supported. Use 7 or later."

### ISSUE THREAD

**Issue by YezCohen** [deptofdefense/AndroidTacticalAssaultKit-CIV#326]
Hi there, 
I'm new to this project, and trying to create the developer environment just so i could compile the app to an .apk file to run on my phone before starting to play with it.

While going over the instructions to build the project on in "[BUILDING.md](https://github.com/deptofdefense/AndroidTacticalAssaultKit-CIV/blob/master/BUILDING.md)", i got to step 3 and runing the first line in the terminal:
`make TARGET=android-arm64-v8a GDAL_USE_KDU=no build_spatialite build_gdal build_commoncommo build_assimp`
it runs, and then gets an error, this is what i got:

> rm -f /h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/data/gdaldata.files
> gdal Java Bindings built
> make -C /h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig BINDINGS=java install
> make[1]: Entering directory '/h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig'
> for dir in java; do (cd $dir; make install) || exit; done
> make[2]: Entering directory '/h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig/java'
> mkdir -p org/gdal/gdal
> mkdir -p org/gdal/gdalconst
> mkdir -p org/gdal/ogr
> mkdir -p org/gdal/osr
> mkdir -p org/gdal/gnm
> if [ -f ".libs/libgdalalljni.so" ] ; then \
> 	cp .libs/*.so . ; \
> fi
> echo ""

> if [ -f ".libs/libgdalalljni.dylib" ] ; then \
> 	cp .libs/*.dylib . ; \
> fi
> ant
> Buildfile: /h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig/java/build.xml
> 
> init:
>      [echo] version = 2.4.4
> 
> compile:
>     [javac] Compiling 58 source files to /h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig/java/build/classes
>     [javac] warning: [options] bootstrap class path not set in conjunction with -source 6
>     [javac] error: Source option 6 is no longer supported. Use 7 or later.
>     [javac] error: Target option 6 is no longer supported. Use 7 or later.
> 
> BUILD FAILED
> /h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig/java/build.xml:22: > Compile failed; see the compiler error output for details.
> 
> Total time: 0 seconds
> GNUmakefile:63: recipe for target 'build' failed
> make[2]: *** [build] Error 1
> make[2]: Leaving directory '/h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig/java'
> GNUmakefile:32: recipe for target 'install' failed
> make[1]: *** [install] Error 2
> make[1]: Leaving directory '/h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/gdal/swig'
> mk/gdal.mk:90: recipe for target '/h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/lib/libgdalalljniso' failed
> make: *** [/h/j/Java-Projects/AndroidTacticalAssaultKit-CIV/takthirdparty/builds/android-arm64-v8a-release/lib/libgdalalljniso] > Error 2

 
does anyone have any idea how to fix it? I feel like i've tried everything and could make it to fix the issue.
I tried to understand where in the code it gets to this but couldnt figure it out:
`[javac] warning: [options] bootstrap class path not set in conjunction with -source 6
    [javac] error: Source option 6 is no longer supported. Use 7 or later.
    [javac] error: Target option 6 is no longer supported. Use 7 or later.`

Any help would be much appreciated!

**Comment by talentedbrute** [comment]
Good day, you do NOT need to compile this ancient code base to be able to develop plugins.  You can obtain the latest SDKs from tak.gov.  It appears that the TAK Product Center has abandoned this project on GitHub.

**Comment by YezCohen** [comment]
> Good day, you do NOT need to compile this ancient code base to be able to develop plugins. You can obtain the latest SDKs from tak.gov, just register for an account. You do not need to be government or military to register.
> 
> It appears that the TAK Product Center has abandoned this project on GitHub.

I see, thank you for the response!

I'll look into the SDKs from tak.gov.

Before I start to develop plugins, I am looking into creating a translation for the app.
It looks like from [this](https://github.com/deptofdefense/AndroidTacticalAssaultKit-CIV/tree/master/atak/ATAK/app/src/main/res) folder that there is just one file of strings to translate in order to translate the app, and that there are languages options for: Arabic, Spanish, French, Japanese and Korean (other than English of course).

Do you have any idea if there is a way to create a translation using the SDKs for the updated code of the app? than I really don’t need to try to compile this repo.

Thank you again for your help!

**Comment by talentedbrute** [comment]
The only way to add a translation is at compile time of the app. 

If you build your own plugin, you can translate the plugin text.

**Comment by YezCohen** [comment]
> The only way to add a translation is at compile time of the app.
> 
> If you build your own plugin, you can translate the plugin text.

I see. Ok thank you!

Anyway - for future reference if anyone gets the same error: The build.xml file has 4 lines (for compile and debug) that calls for "source" and "option" with the value "1.6", even when I changed it manually to "7", because while compiling it it called this file multiple times, and every time it changed it back to "1.6", so the next time it called it - it raised an error again.
To fix it, I ran a python script in the background that monitored the build.xml file, and every time it changed to 1.6, it changed it back to 7.
